### PR TITLE
updated the images and markdown showing wrong file names of COBOL compilation output

### PR DESCRIPTION
**Pull request by ashiskumarnaik**
## Proposed changes

updated the images and markdown showing wrong file names of COBOL compilation output

Fixes # (#301 )

## Type of change

What type of changes does your PR introduce to the COBOL Programming Course? _Put an `x` in the boxes that apply._

- [x] Bug fix (change which fixes one or more issues)
- [ ] New feature (change which adds functionality or features to the course)
- [ ] Translations (change which adds or modifies translations of the course)
- [ ] Documentation (change which modifies documentation related to the course)
- [ ] This change requires an update to the course's z/OS environment

## Checklist:

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [Contributing Guideline](https://github.com/openmainframeproject/cobol-programming-course/blob/master/CONTRIBUTING.md)
- [x] I have included a title and description for this PR
- [x] I have DCO-signed all of my commits that are included in this PR
- [x] I have tested it manually and there are no regressions found
- [ ] I have commented my code, particularly in hard-to-understand areas (if appropriate)
- [ ] I have made corresponding changes to the documentation (if appropriate)

### CODE REVIEW

File: COBOL Programming Course #2 - Learning COBOL/Images/image104.png
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/Images/image104.png:1]: Can we remove all reference of your Z9XXXX ID from the image? We don't want people to start trying out your ID on the system and make the system blocks your ID :)

File: COBOL Programming Course #2 - Learning COBOL/Images/image106.png
Comment (tanto259) [inline COBOL Programming Course #2 - Learning COBOL/Images/image106.png:1]: Same thing here with the ID

**Review by tanto259 — CHANGES_REQUESTED** [review]
Thanks again, Ashis! I did not realize that they change the layout of the Jobs view on Zowe Explorer again...

Left some comments on 2 of the images.

**Comment by ashiskumarnaik** [comment]
Hi @tanto259  If I take screenshots , then my user id will also be captured . Do I need to blur it out ? (any suggestion)

**Pull request closed without merging**